### ISSUE TITLE
.vue files

### ISSUE THREAD

**Issue by archaistvolts** [arijs/prettier-with-tabs#28]
This doesn't seem to be prettying my .vue files.  Is this possible?  If so, what configuration must I set to set to make it work? 

These don't seem to work: 
```javascript
  "prettier.javascriptEnable": [
    "javascript",
    "javascriptreact",
    "vue"
  ],
  "prettier.cssEnable": [
    "css",
    "less",
    "scss",
    "postcss",
    "vue"
  ]
```

**Comment by remimarsal** [comment]
@travisstaloch where does this configuration come from ?

**Comment by archaistvolts** [comment]
Oops I should have mentioned this is from vscode user settings file.  Adding "vue" was only a shot in the dark after the prettier configurations didn't produce any effect.

**Comment by remimarsal** [comment]
Alright, what extension are you using?

**Comment by archaistvolts** [comment]
Prettier Now

**Comment by remimarsal** [comment]
Great! Did you install any extension to detect Vue files?

**Comment by archaistvolts** [comment]
Thank you. Yes, I've been using Vetur for a bit.  I had been using it with Prettier - JavaScript formatter for sometime and it was working.  I changed because Prettier Now offered more configuration options.

**Comment by remimarsal** [comment]
Yes the other one seems to be discontinued. Ok I'll give Vetur a shot and see what's happening. I actually never tried the extension on a Vue file!

**Comment by remimarsal** [comment]
Ok the issue is not the language ID which is indeed `vue`. Vue syntax is not **yet** supported by the official Prettier. (https://github.com/prettier/prettier/issues/2097)

**Comment by archaistvolts** [comment]
Oh I see.  Well thanks a lot for looking!  So this project must depend on prettier then.  Hope they get vue support added soon.

**Comment by remimarsal** [comment]
You're welcome! :wink:

**Comment by soccer99** [comment]
Vue is now supported on Prettier, can you add the support here?
